src/NzbDrone.Core/Datastore/Migration/013_email_multiple_addresses.cs

```diff
@@ -0,0 +1,97 @@
+using System;
```

Comment:
To seems to already be a string[] on Readarr? If so we don't need the migration

Ref.
https://github.com/Readarr/Readarr/blob/8406dbe49de5b06e0ec8dfad46c1b2575c4dff74/src/NzbDrone.Core/Notifications/Email/EmailSettings.cs#L52